apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -209,57 +194,6 @@ export class SendMessageEmail extends SendMessageBase {
       })
     );
 
-    const customTemplate = SendMessageEmail.addPreheader(content as string, emailChannel.template.contentType);
-    let html;
-    try {
-      html = await this.compileTemplate.execute(
-        CompileTemplateCommand.create({
-          templateId: isEditorMode ? 'basic' : 'custom',
-          customTemplate: customTemplate,
-          data: {
-            subject,
-            preheader,
-            branding: {
-              logo: organization.branding?.logo,
-              color: organization.branding?.color || '#f47373',
-            },
-            blocks: isEditorMode ? content : [],
-            step: {
-              digest: !!command.events?.length,
-              events: command.events,
-              total_count: command.events?.length,
-            },
-            ...command.payload,
-          },
-        })
-      );
-    } catch (error) {
-      await this.sendErrorStatus(
-        message,
-        'error',
-        'mail_unexpected_error',
-        'syntax error in email editor',
-        command,
-        notification,
-        LogCodeEnum.SYNTAX_ERROR_IN_EMAIL_EDITOR
-      );
-
-      await this.createExecutionDetails.execute(
-        CreateExecutionDetailsCommand.create({
-          ...CreateExecutionDetailsCommand.getDetailsFromJob(command.job),
-          detail: DetailEnum.MESSAGE_CONTENT_SYNTAX_ERROR,
```

Comment:
should we remove this var: MESSAGE_CONTENT_SYNTAX_ERROR ?

Reply:
It was a bit redundant, as we already have it [here](https://github.com/novuhq/novu/blob/fd21da7a1a62a70aaba67a09c1a842417c330fe5/apps/api/src/app/events/usecases/send-message/send-message.base.ts#L49)

I now  generate the html inside `compileEmailTemplateUsecase`, any syntax error will be caught